packages/background/src/keyring/keyring.ts

```diff
@@ -1155,6 +1155,10 @@ export class KeyRing {
   ): Promise<{
     multiKeyStoreInfo: MultiKeyStoreInfoWithSelected;
   }> {
+    if (this.password !== "" && this.status !== KeyRingStatus.UNLOCKED) {
```

Comment:
@AbbasAliLokhandwala Changes are no longer required, as the issue is only reproducible for accounts with auto-lock enabled. Therefore, please revert the keyring-related modifications.

Fix: https://github.com/fetchai/asi-alliance-wallet/pull/422

cc: @agent-dominatrix

Reply:
Changes resolved